### PR TITLE
Add instructions for Recent policy, minor formatting

### PR DESCRIPTION
**Pull request by s-ben**
Added instructions for Recent (`recentWeeklyDecayRate`) policy. paired-with @KuraFire (reused their example and wording from Discord help message). 

Added example `grain.json` file. 

Capitalized several instances of cred->Cred.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,16 +91,34 @@ Run `yarn clean-all` if the `yarn start` command fails due to a change in the co
...
-with grain relative to their cred scores, that people will receive less grain in the balanced distribution while people
-who have been "underpaid" relative to their cred will receive more grain.
+The `recentPerWeek` policy weights contributions with an exponential decay. The `recentWeeklyDecayRate` parameter determines the exponential rate of decay. The weighting for each week is equal to (1 - `recentWeeklyDecayRate`) raised to the power of the week number. For example, if `recentWeeklyDecayRate` is set to 1, this policy is equivalent to the `immediatePerWeek` policy; 100% of grain is distributed based on Cred generated in the last week and Cred created in prior weeks is "discounted" to 0%. If `recentWeeklyDecayRate` is set to 0.5 (i.e. 50% discount), as in the above example, the policy will count 100% of the Cred generated in the last week, 50% of the Cred generated in the week before, 25% from the week before that, 12.5% from the week before that, and so on. In other words, `recentPerWeek` is a lot more flexible in how it rewards the most recent contributions without making it a hard 1-week cut-off.
```
Comment (elihanover) [inline README.md:121]: This is necessary information, but I tend to prefer README's as short as possible, in this case I think of it as the minimum information needed to get an instance running.  I suggest instead linking to our docs on the details of these policies with a line like:

### FAQ:
- [What does each policy do?]()
- [How should I choose the right policy for me?]()
- [Can I change the policies in the future?]()

File: README.md [hunk 2]
```diff
@@ -91,16 +91,34 @@ Run `yarn clean-all` if the `yarn start` command fails due to a change in the co
 - `yarn grain` distributes Grain according to the current Cred scores, and the config in `config/grain.json`. 
 
 This repo also contains a GitHub action for automatically distributing grain. It will run every Sunday and create a Pull Request
-with the ledger updated with the new grain balances based on the users cred scores. The amount of grain to get distributed
-every week can be defined in the `config/grain.json` file. There are two different policies that can be used to control
+with the ledger updated with the new grain balances based on the users Cred scores. The amount of grain to get distributed
+every week can be defined in the `config/grain.json` file.
```
Comment (elihanover) [inline README.md:95]: This should be after descriptions of the policies IMO.  Start with a high level picture of distributing grain, and then go into details of how it works.

File: README.md (hunk 1 above)
Comment (KuraFire) [reply README.md:121]: +1 to this.

**Review by KuraFire — CHANGES_REQUESTED** [review]
I favor single-sourcing-of-truth as much as possible everywhere we can't reference & import content blocks (like in Roam or Obsidian). Linking to the documents instead of copy-pasting would be strongly preferred.

**Comment by decentralion** [comment]
Removing myself as a reviewer, I trust @elihanover and @KuraFire have it covered :)

File: README.md (hunk 1 above)
Comment (s-ben) [reply README.md:121]: Agreed. @elihanover, when your PR adding this info to the Grain page of the docs is merged, I'll implement your above suggested change. Just left a quick [review on your PR](https://github.com/sourcecred/docs/pull/250#pullrequestreview-564682249).

**Comment by s-ben** [comment]
@KuraFire, @elihanover, have removed text descriptions of policies, added text linking back to Grain page on docs. PR looks good to me now.

Have left the example `grain.json` config, as I see that wasn't ported over to the Grain page, and I think it makes sense here.

**Review by KuraFire — APPROVED** [review]
Looks good 👍🏼

**Pull request merged**